components/raftstore/src/store/worker/pd.rs

```diff
@@ -36,11 +36,13 @@ use crate::store::{CasualMessage, PeerMsg, RaftCommand, RaftRouter, StoreMsg};
 
 use collections::HashMap;
 use concurrency_manager::ConcurrencyManager;
+use futures::FutureExt;
 use pd_client::metrics::*;
 use pd_client::{Error, PdClient, RegionStat};
 use tikv_util::metrics::ThreadInfoStatistics;
 use tikv_util::time::UnixSecs;
 use tikv_util::worker::{FutureRunnable as Runnable, FutureScheduler as Scheduler, Stopped};
+use tokio::time::delay_for;
```

Comment:
Tokio's timer needs it's own reactor. I'm afraid it doesn't work here. It would be a better choice to use `GLOBAL_TIMER_HANDLE`.